ONSdigital/eq-author-app
# Ear 204 scroll to top

### What is the context of this PR?

If you had several long pages and were scrolled to the bottom (or down) on one, moving to another page would keep you at that scroll location instead of delivering you to the top of the new page. This PR introduces a util which wraps around the entire editor pane which scrolls to the top of a page when you visit a new one.

### How to review 

Create two long pages (so the scroll bar appears on each). Scroll to the bottom of one and move to the other page. You should be delivered to the top of the new page.

* Import the **Capability examples** questionnaire from pre-prod Author into your local environment and:
    * ensure it can be opened in Author;
    * then, ensure it can be viewed in Runner by pressing the **view survey** button
* Does this require a migration? We need one if existing JSON schema properties change

### What to do after everything is green

1. * [ ] Bring this branch up-to-date with Origin/Master
2. * [ ] If there are a lot of commits or some are not helpful to read, squash them down
3. * [ ] Click the **Merge** button on this pull request
4. * [ ] Does this change mean the **Capability examples** questionnaire should be updated?
5. * [ ] Move the Trello card for this task into the next stage of the process
